### PR TITLE
What's a Namespace Package?

### PR DESCRIPTION
**Pull request by iansedano**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: namespace-package/plugins/data-repo-plugs/data_repo/read.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            continue
+        try:
+            read_function = import_module(
+                f".{item.stem}", f"{__package__}.readers"
+            ).read
+            if callable(read_function):
+                readers[item.stem] = read_function
+        except ImportError:
+            continue
+        except AttributeError as e:
```
Comment (gahjelle) [inline namespace-package/plugins/data-repo-plugs/data_repo/read.py:18]: This `e` isn't used and can be removed: `except AttributeError:`

File: namespace-package/business/snake-corp-dateutil/snake_corp/dateutil.py [hunk 2]
```diff
@@ -5,10 +5,8 @@ def days_to_snake_day(from_date=None):
...
-        return 0
-    if from_date.month >= 7 or (from_date.month == 7 and from_date.day >= 16):
-        day_of_snake = datetime(from_date.year + 1, 8, 16)
+    if from_date.month > 7 or (from_date.month == 7 and from_date.day > 16):
+        day_of_snake = datetime(from_date.year + 1, 7, 16)
     else:
-        day_of_snake = datetime(from_date.year, 8, 16)
-    return (day_of_snake - from_date).days
+        day_of_snake = datetime(from_date.year, 7, 16)
+    return (day_of_snake - from_date).days
```
Comment (gahjelle) [inline namespace-package/business/snake-corp-dateutil/snake_corp/dateutil.py:12]: The linter complains about the lack of a newline at the end of this file

**Review by gahjelle — COMMENTED** [review]
Almost there! Just a few quibbles to fix.

File: namespace-package/plugins/data-repo-plugs/data_repo/read.py (hunk 1 above)
Comment (iansedano) [reply namespace-package/plugins/data-repo-plugs/data_repo/read.py:18]: Updated here and in the article

File: namespace-package/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+You have the Hello World example, and the source code for the Snake Shop Corporation example that's hosted on PyPI. Aditionally, as an extra, you have the code to generate the fake data used.
+
+You need to install every subpackage to get it working properly.
+
+## Examples
+
+In this repository you'll find four different examples showing ways namespace packages work or can be used.
+
+### Hello World
```
Comment (KateFinegan) [inline namespace-package/README.md:13]: @iansedano @gahjelle Is Hello World still present in the repository? I don't see it, and it's not mentioned in the tutorial.

**Pull request merged**